renovate.json

```diff
@@ -2,12 +2,12 @@
   "extends": [
     "config:base"
   ],
-  "schedule": ["on Monday every 2 weeks"],
+  "schedule": ["before 8am on the first day of the month"],
```

Comment:
Maybe we should do a less predictable day?

Reply:
scheduling for 3rd day of the month